jwst/associations/lib/rules_level2b.py

```diff
@@ -541,3 +541,33 @@ def finalize(self):
 
         """
         return self.make_nod_asns()
+
+
+@RegistryMarker.rule
+class Asn_Lv2WFSC(
+        DMSLevel2bBase
+):
+    """Level2b Wavefront Sensing & Controle"""
```

Comment:
You say "potatoe", I say "potato"? I think the word you're looking for is "Control".

Reply:
👍